### PR TITLE
Fix Prepackaged Server nil and update core examples

### LINKED ISSUE #1622: Explainer example notebook use v1alpha2 CRDS
* Also add annotation for grpc setup not needed for REST examples

### PR DESCRIPTION
**Pull request by cliveseldon**
Fixes #1622

 * Fix nil when creating prepackaged server deployments
 * Update core examples due to new deployment names

### CODE REVIEW

**Comment by seldondev** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by:
To fully approve this pull request, please assign additional approvers.
We suggest the following additional approver: **cliveseldon**

If they are not already assigned, you can assign the PR to them by writing `/assign @cliveseldon` in a comment when ready.

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=SeldonIO%2Fseldon-core).

The pull request process is described [here](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process)

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/SeldonIO/seldon-core/blob/master/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Comment by seldondev** [comment]
Fri Mar 27 15:51:14 UTC 2020
The logs for [pr-build] [1] will show after the pipeline context has finished.  
https://github.com/SeldonIO/seldon-core/blob/gh-pages/jenkins-x/logs/SeldonIO/seldon-core/PR-1623/1.log

impatient try  
jx get build logs SeldonIO/seldon-core/PR-1623 --build=1

**Comment by seldondev** [comment]
Fri Mar 27 15:51:24 UTC 2020
The logs for [lint] [2] will show after the pipeline context has finished.  
https://github.com/SeldonIO/seldon-core/blob/gh-pages/jenkins-x/logs/SeldonIO/seldon-core/PR-1623/2.log

impatient try  
jx get build logs SeldonIO/seldon-core/PR-1623 --build=2

File: notebooks/helm_examples.ipynb [hunk 1]
```diff
@@ -246,13 +246,11 @@
...
    "execution_count": null,
-   "metadata": {
-    "scrolled": true
-   },
+   "metadata": {},
    "outputs": [],
    "source": [
-    "!kubectl rollout status deploy/myabtest-default-0\n",
-    "!kubectl rollout status deploy/myabtest-default-1"
+    "!kubectl rollout status deploy/$(kubectl get deploy -l seldon-deployment-id=myabtest -o jsonpath='{.items[0].metadata.name}')\n",
```
Comment (RafalSkolasinski) [inline notebooks/helm_examples.ipynb]: Is there a reason why a better looking previous syntax cannot work anymore?

**Comment by seldondev** [comment]
Fri Mar 27 17:20:40 UTC 2020
The logs for [integration] [3] will show after the pipeline context has finished.  
https://github.com/SeldonIO/seldon-core/blob/gh-pages/jenkins-x/logs/SeldonIO/seldon-core/PR-1623/3.log

impatient try  
jx get build logs SeldonIO/seldon-core/PR-1623 --build=3

**Comment by seldondev** [comment]
@cliveseldon: The following test **failed**, say `/retest` to rerun them all:

Test name | Commit | Details | Rerun command
--- | --- | --- | ---
integration | 1a36cd423ba1b0ae9c3ac79ae90774dd1193fb31 | [link]() | `/test integration`



<details>

Instructions for interacting with me using PR comments are available [here](https://git.k8s.io/community/contributors/guide/pull-requests.md).  If you have questions or suggestions related to my behavior, please file an issue against the [kubernetes/test-infra](https://github.com/kubernetes/test-infra/issues/new?title=Prow%20issue:) repository. I understand the commands that are listed [here](https://go.k8s.io/bot-commands).
</details>

**Comment by seldondev** [comment]
Fri Mar 27 18:01:09 UTC 2020
The logs for [integration] [4] will show after the pipeline context has finished.  
https://github.com/SeldonIO/seldon-core/blob/gh-pages/jenkins-x/logs/SeldonIO/seldon-core/PR-1623/4.log

impatient try  
jx get build logs SeldonIO/seldon-core/PR-1623 --build=4

**Comment by seldondev** [comment]
Fri Mar 27 22:58:03 UTC 2020
The logs for [lint] [6] will show after the pipeline context has finished.  
https://github.com/SeldonIO/seldon-core/blob/gh-pages/jenkins-x/logs/SeldonIO/seldon-core/PR-1623/6.log

impatient try  
jx get build logs SeldonIO/seldon-core/PR-1623 --build=6

**Comment by seldondev** [comment]
Fri Mar 27 22:59:18 UTC 2020
The logs for [integration] [7] will show after the pipeline context has finished.  
https://github.com/SeldonIO/seldon-core/blob/gh-pages/jenkins-x/logs/SeldonIO/seldon-core/PR-1623/7.log

impatient try  
jx get build logs SeldonIO/seldon-core/PR-1623 --build=7

**Comment by seldondev** [comment]
Fri Mar 27 22:59:46 UTC 2020
The logs for [pr-build] [5] will show after the pipeline context has finished.  
https://github.com/SeldonIO/seldon-core/blob/gh-pages/jenkins-x/logs/SeldonIO/seldon-core/PR-1623/5.log

impatient try  
jx get build logs SeldonIO/seldon-core/PR-1623 --build=5

**Comment by seldondev** [comment]
Fri Mar 27 22:59:51 UTC 2020
The logs for [notebooks] [8] will show after the pipeline context has finished.  
https://github.com/SeldonIO/seldon-core/blob/gh-pages/jenkins-x/logs/SeldonIO/seldon-core/PR-1623/8.log

impatient try  
jx get build logs SeldonIO/seldon-core/PR-1623 --build=8

**Comment by seldondev** [comment]
Sat Mar 28 08:01:09 UTC 2020
The logs for [integration] [9] will show after the pipeline context has finished.  
https://github.com/SeldonIO/seldon-core/blob/gh-pages/jenkins-x/logs/SeldonIO/seldon-core/PR-1623/9.log

impatient try  
jx get build logs SeldonIO/seldon-core/PR-1623 --build=9

**Comment by seldondev** [comment]
Sat Mar 28 09:59:54 UTC 2020
The logs for [pr-build] [10] will show after the pipeline context has finished.  
https://github.com/SeldonIO/seldon-core/blob/gh-pages/jenkins-x/logs/SeldonIO/seldon-core/PR-1623/10.log

impatient try  
jx get build logs SeldonIO/seldon-core/PR-1623 --build=10

**Comment by seldondev** [comment]
Sat Mar 28 09:59:58 UTC 2020
The logs for [lint] [11] will show after the pipeline context has finished.  
https://github.com/SeldonIO/seldon-core/blob/gh-pages/jenkins-x/logs/SeldonIO/seldon-core/PR-1623/11.log

impatient try  
jx get build logs SeldonIO/seldon-core/PR-1623 --build=11

**Comment by seldondev** [comment]
Sat Mar 28 10:59:01 UTC 2020
The logs for [integration] [12] will show after the pipeline context has finished.  
https://github.com/SeldonIO/seldon-core/blob/gh-pages/jenkins-x/logs/SeldonIO/seldon-core/PR-1623/12.log

impatient try  
jx get build logs SeldonIO/seldon-core/PR-1623 --build=12

**Comment by seldondev** [comment]
[APPROVALNOTIFIER] This PR is **APPROVED**

This pull-request has been approved by: *<a href="https://github.com/SeldonIO/seldon-core/pull/1623#issuecomment-605453160" title="Approved">cliveseldon</a>*

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=SeldonIO%2Fseldon-core).

The pull request process is described [here](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process)

<details >
Needs approval from an approver in each of these files:

- ~~[OWNERS](https://github.com/SeldonIO/seldon-core/blob/master/OWNERS)~~ [cliveseldon]

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Comment by seldondev** [comment]
Sat Mar 28 14:17:29 UTC 2020
The logs for [pr-build] [13] will show after the pipeline context has finished.  
https://github.com/SeldonIO/seldon-core/blob/gh-pages/jenkins-x/logs/SeldonIO/seldon-core/PR-1623/13.log

impatient try  
jx get build logs SeldonIO/seldon-core/PR-1623 --build=13

**Comment by seldondev** [comment]
Sat Mar 28 14:17:35 UTC 2020
The logs for [lint] [14] will show after the pipeline context has finished.  
https://github.com/SeldonIO/seldon-core/blob/gh-pages/jenkins-x/logs/SeldonIO/seldon-core/PR-1623/14.log

impatient try  
jx get build logs SeldonIO/seldon-core/PR-1623 --build=14

**Comment by seldondev** [comment]
@cliveseldon: The following test **failed**, say `/retest` to rerun them all:

Test name | Commit | Details | Rerun command
--- | --- | --- | ---
notebooks | 23deaa3731b4d2e9b64e4f992a967d134ff47fe6 | [link]() | `/test notebooks`



<details>

Instructions for interacting with me using PR comments are available [here](https://git.k8s.io/community/contributors/guide/pull-requests.md).  If you have questions or suggestions related to my behavior, please file an issue against the [kubernetes/test-infra](https://github.com/kubernetes/test-infra/issues/new?title=Prow%20issue:) repository. I understand the commands that are listed [here](https://go.k8s.io/bot-commands).
</details>

**Pull request merged**